cypress/integration/portal-dashboard/portal-dashboard-response-table.spec.js

```diff
@@ -25,46 +25,85 @@ context("Portal Dashboard Response Table",()=>{
       .find('[data-cy=NotStarted]').should('be.visible');
   });
 
+  function aType(name) {
+    // TODO: Figure out how to import AnswerTypes so we can do the following:
+    // const foundType = AnswerTypes.find(at => at.name === name);
+    // if (!foundType) {
+    //   throw new Error(`Invalid answer type: ${name}`);
+    // }
+    return {name};
+  }
+
+  function getAnswerIconId (answerType) {
+    const searchRegExp = / /g;
+    const iconId = answerType ? answerType.name.toLowerCase().replace(searchRegExp, "-") : "";
+    return iconId;
+  }
+
+  function aTypeMatcher(answerType) {
+    const dataCy = answerType === null ?
+      "no-answer" :
+      getAnswerIconId(answerType);
+    return `[data-cy=${dataCy}]`;
+  }
+
+  function checkAnswerTable(answerTable) {
+    const numRows = answerTable.length;
+    const numCols = answerTable[0].length;
+    for (let row = 0; row < numRows; row++){
+      for (let col = 0; col < numCols; col++){
+        const expectedAnswerType = answerTable[row][col];
+        cy.get('[data-cy=student-answers-row]')
+        .eq(row)
+        .find('[data-cy=student-answer]')
+        .eq(col)
+        .find(aTypeMatcher(expectedAnswerType)).should('be.visible');
+      }
+    }
+  }
+
   it('verify we display the correct student answer icons',()=>{
+    const OR = aType("Open Response Completed");
+    const MC = aType("Multiple Choice NonScored Completed");
+    const Mx = aType("Multiple Choice Scored Incorrect");
+    const Mv = aType("Multiple Choice Scored Correct");
+    const IQ = aType("Image Question with Open Response Completed");
+    const IA = aType("Interactive Completed");
```

Comment:
It seems like the choice of `IA` for the variable name was intentional even though I would've expected `IC`. I'm not sure what the A indicates, but I'm probably just not familiar with whatever it is.

Reply:
All of them are "Complete", so it seemed weird to add a C to this one when I didn't add one other places. The C in MC was for Choice.  The A was for "active". But I agree it is weird. I think I've seen Piotr use IA for interactives before. But if you have another 2 letter suggestion for Interactive I'm open to it.

I've got one more portal-report change that I'll open a new PR so I can apply your suggestion there.

Reply:
Maybe `IN`?

Reply:
Yeah, that seems reasonable. But reading your explanation and looking at the code again, I think `IA` makes sense, too. The only other alternatives I can think of are `ID` ("Interactive Done") or `IF` ("Interactive Finished") but those seem potentially confusing, and you're using "Completed" elsewhere. I think it's fine as is, or if you switch it to `IN`.

Reply:
OK I'll just leave it then, easier than changing it. 🤷